src/configure.py

```diff
@@ -341,65 +345,65 @@ def load_arguments():
-    parser.add_argument( "--flu_scheme", type=str, metavar="SCHEME", gamer_name="FLU_SCHEME",
+    parser.add_argument( "--flu_scheme", type=str, metavar="TYPE", gamer_name="FLU_SCHEME",
                          default="CTU", choices=["RTVD", "MHM", "MHM_RP", "CTU"],
                          depend={"model":"HYDRO"},
                          constrain={ "RTVD":{"unsplit_gravity":False, "passive":0, "dual":NONE_STR, "eos":"GAMMA"},
                                      "CTU":{"eos":"GAMMA"} },
-                         help="The fluid solver for HYDRO model. MHD only supports MHM_RP and CTU.\n"
+                         help="The hydrodynamic/MHD integrator. MHD only supports MHM_RP and CTU.\n"
                        )
 
     parser.add_argument( "--slope", type=str, metavar="TYPE", gamer_name="LR_SCHEME",
                          default="PPM", choices=["PLM", "PPM"],
                          depend={"model":"HYDRO"},
-                         help="Select the spatial data reconstruction method.\n"
+                         help="The spatial data reconstruction method (PLM: piecewise-linear, PPM: piecewise-parabolic). Useless for <--flu_scheme=RTVD>.\n"
                        )
 
     parser.add_argument( "--flux", type=str, metavar="TYPE", gamer_name="RSOLVER",
                          default="ROE", choices=["EXACT", "ROE", "HLLE", "HLLC", "HLLD"],
                          depend={"model":"HYDRO"},
                          constrain={ "ROE":{"eos":"GAMMA"},
                                      "EXACT":{"eos":"GAMMA"} },
-                         help="The Riemann solver. Pure hydro: EXACT/ROE/HLLE/HLLC, MHD: ROE/HLLE/HLLD. Useless for RTVD.\n"
+                         help="The Riemann solver. Pure hydro: EXACT/ROE/HLLE/HLLC^, MHD: ROE/HLLE/HLLD^ (^ indicates the recommended solvers). Useless for RTVD.\n"
                        )
 
     parser.add_argument( "--dual", type=str, metavar="TYPE", gamer_name="DUAL_ENERGY",
                          default=NONE_STR, choices=[NONE_STR, "DE_ENPY", "DE_EINT"],
                          depend={"model":"HYDRO"},
                          constrain={ "DE_ENPY":{"eos":"GAMMA"} },
-                         help="The dual-energy formalism. DE_ENPY: entropy, DE_EINT: internal energy. DE_EINT is not supported yet. Useless for RTVD.\n"
+                         help="The dual-energy formalism (DE_ENPY: entropy, DE_EINT: internal energy). DE_EINT is not supported yet. Useless for RTVD.\n\n"
                        )
 
     parser.add_argument( "--mhd", type=str2bool, metavar="BOOLEAN", gamer_name="MHD",
                          default=False,
                          depend={"model":"HYDRO"},
                          constrain={ True:{"flu_scheme":["MHM_RP", "CTU"], "flux":["EXACT", "ROE", "HLLE", "HLLD"]},
-                                     False:{"flux":["EXACT", "ROE", "HLLE", "HLLC"]} },
+                                     False:{"flux":["ROE", "HLLE", "HLLC"]} },
```

Comment:
I thought we should remove `EXACT` from the `True` list and add it back to the `False` list.